libs/designer/src/lib/ui/configuretemplate/templateprofile/manifestform.tsx

```diff
@@ -30,6 +33,17 @@ export const TemplateManifestForm = () => {
 
   return (
     <div className="msla-templates-wizard-tab-content" style={{ width: '70%', marginTop: '8px' }}>
+      <DescriptionWithLink
+        text={intl.formatMessage({
+          defaultMessage:
+            'Add details to help template users evaluate this template. The profile includes the information shown to users and settings that control how the template is filtered and displayed.',
+          id: 'tkkN++',
+          description: 'Description for template profile tab',
+        })}
+        linkText={resources.LearnMore}
+        linkUrl="https://learn.microsoft.com/en-us/azure/logic-apps/logic-apps-templates-manifest"
```

Comment:
NIT: Is this the right link? It redirects to a 404, I guess it hasn't been published, but wanted to double check

Reply:
Yes it is not... Divya will get us the link later.. these are just placeholders